htdocs/install/mysql/tables/llx_paiementcharge.sql

```diff
@@ -19,7 +19,9 @@
 create table llx_paiementcharge
 (
   rowid           integer AUTO_INCREMENT PRIMARY KEY,
+  entity		  integer DEFAULT 1,
```

Comment:
We should not need a field entity here, because a payment of a social contribution is linked to a social contribution.
And the social contribution is already inside one company (so the field entity is already inside the table llx_chargesociales)
And if a social contribution is inside entity X, the payment must also be into entity X. So we are alreay able to know the entity of a payement using the entity of its parent social contribution.